src/components/selector-dropdown/SelectorDropdown.js

```diff
@@ -311,13 +312,13 @@ class SelectorDropdown extends React.Component<Props, State> {
                 onPaste={this.handleInput}
                 ref={this.selectorDropdownRef}
             >
-                {React.cloneElement(selector, { inputProps })}
-                {isOpen && (
-                    <div className={`${OVERLAY_WRAPPER_CLASS} is-visible`}>
+                <PopperComponent placement={PLACEMENT_BOTTOM_START} isOpen={isOpen}>
```

Comment:
It will have the same behavior as the SelectorField elements above - it render on top of the triggering element. There is a min/max height and once it crosses that threshold, it will begin to be clipped by the viewport. This is entirely dependent on the individual max height, but so far I don't see any issues on "Small" viewport sizes (i.e. 400px height)

For example, at under 400px height, you already run into other issues. I guess we don't expect the best experience under 400px height in general
<img width="506" alt="image" src="https://user-images.githubusercontent.com/1041516/222004701-1cb8d2a4-b72a-46f8-a6d4-0176bc08b834.png">